docs/Site-Reliability-Guardian/SRGAutomation.md

```diff
@@ -45,7 +45,7 @@ The optional values are sent as part of the BizEvent into Dynatrace and can be u
 | --timespan        | SRG_EVALUATION_TIMESPAN        | Grab the last X minutes of data for the evaluation (default: "5"). This conflicts with start-time and end-time so only one can be set at once |
 | --application     | SRG_EVALUATION_APPLICATION     | This is the application or service name. default: ""                                                                                          |
 | --provider        | SRG_EVALUATION_PROVIDER        | Provider of the request. i.e. github, jenkins, jenkins-production-1 etc. default: "cicd"                                                      |
-| --version         | SRG_APP_VERSION                | Version of the app. i.e. v1.0.1. default: ""                                                                                                  |
+| --release-version | SRG_APP_VERSION                | Version of the app. i.e. v1.0.1. default: ""                                                                                                  |
```

Comment:
Nice to have, but the other params start with SRG_EVALUATION; it might be good to continue with this standard for this param as well.